### PR TITLE
Correct radioblog and homepage synopsis text + more

### PR DESCRIPTION
**Pull request by dchan3**
Require synopses in posts and update deprecated dependency

Revise synopsis render functions

### CODE REVIEW

**Comment by dchan3** [comment]
I shall see if the `moment-timezone` reinstallation and subsequent refactoring addresses #70.

File: client/main.css [hunk 1]
```diff
@@ -2858,3 +2885,54 @@ margin-bottom: 0 !important;
...
+}
+@media screen and (min-width: 430px) and (max-width: 567px) {
+  .banner-container {
+    font-size: 14px;
+  }
+}
+@media screen and (max-width: 429px) {
+  .banner-container {
+    font-size: 11.5px;
+  }
```
Comment (daveynight) [inline client/main.css]: I can't accept this and line 2928. Nicki painstakingly created a type system, and using whatever font sizes outside of it breaks that. See lines 35-40 for acceptable sizes.

File: client/main.js [hunk 2]
```diff
@@ -36,7 +36,8 @@ Meteor.startup(function () {
 
 AutoForm.addHooks(['partyForm'],{
   onSuccess: function(formType, result) {
-    console.log("Successfully submitted partyForm!\nthis.docId = " + this.docId);
+    console.log('Successfully submitted partyForm!\nthis.docId = ' +
+      this.docId);
```
Comment (daveynight) [inline client/main.js]: Please remove console logging statements if they're not absolutely needed.

File: client/main.js [hunk 3]
```diff
@@ -45,9 +46,9 @@ Meteor.subscribe('bannedProfiles');
 
 Tracker.autorun(() => {
   if (Meteor.loggingIn() || Meteor.user()) {
-    if (Profiles.findOne({userId: Meteor.userId()}) !== undefined) {
-      if (Profiles.findOne({userId: Meteor.userId()}).banned) {
-        throwError("Login denied. This account is currently disabled.");
+    if (Profiles.findOne({ userId: Meteor.userId() }) !== undefined) {
+      if (Profiles.findOne({ userId: Meteor.userId() }).banned) {
```
Comment (daveynight) [inline client/main.js]: Why the nested if statements here? It's a bad pattern. Preferably we'd join these with `&&`.

Comment (daveynight) [reply client/main.js]: Maybe there's a reason to do it this way but IDK

File: imports/ui/components/application/layout.html [hunk 4]
```diff
@@ -1,4 +1,15 @@
 <template name='layout'>
+  {{#if Template.subscriptionsReady}}
+    {{#if home}}
+      {{#if notice}}
```
Comment (daveynight) [inline imports/ui/components/application/layout.html:4]: I wish we could get rid of these too. Handlebars doesn't have an && huh.

**Review by daveyshindig — CHANGES_REQUESTED** [review]
Wow that was a lot of formatting changes. Thanks for all the enhancements and so on. I've requested a few changes. Check the comments.

**Review by daveyshindig — APPROVED** [review]

**Pull request merged**